README.md

```diff
@@ -16,17 +16,15 @@
 
 `Routerify` provides a lightweight, idiomatic, composable and modular router implementation with middleware support for the Rust HTTP library [hyper](https://hyper.rs/).
 
-`Routerify` offers the following features:
-
-- 📡 Allows defining complex routing logic.
-- 🔨 Provides middleware support.
-- 🌀 Supports Route Parameters.
-- 🚀 Fast as it's using [`RegexSet`](https://docs.rs/regex/1.4.3/regex/struct.RegexSet.html) to match routes. 
-- 🍺 It supports any response body type as long as it implements the [HttpBody](https://docs.rs/hyper/0.14.4/hyper/body/trait.HttpBody.html) trait.
-- ❗ Provides a flexible error handling strategy.
-- 💁 Provides `WebSocket` [support](https://github.com/routerify/routerify-websocket) out of the box.
-- 🔥 Allows data/state sharing across the route and middleware handlers.
-- 🍗 Exhaustive [examples](https://github.com/routerify/routerify/tree/master/examples) and well documented.
+Routerify's core features:
+
+- 🌀 Design complex routing using **scopes** and **middlewares**
```

Comment:
Do we need to emphasize with bold here?

Reply:
Fair question, I thought it served a purpose but I can see that's non-obvious. Let me try one better...

```suggestion
- 🌀 Design complex routing using [scopes](https://github.com/routerify/routerify/blob/update-readme-features/examples/scoped_router.rs) and [middlewares](https://github.com/routerify/routerify/blob/update-readme-features/examples/middleware.rs)
```

Thoughts?

Reply:
Maybe point to the docs instead of source, e.g. https://docs.rs/routerify/#scopingmounting-router and https://docs.rs/routerify/#middleware

And also change the last line to include link to docs.rs/routerify when mentioning documentation, next to examples.

Reply:
In my experience when trying to learn a few other crates, their documentation for examples linked directly to source, e.g. see [warp's examples](https://github.com/seanmonstar/warp/tree/master/examples) + [link in top-level README](https://github.com/seanmonstar/warp#example), along with [Gotham's examples](https://github.com/gotham-rs/gotham/tree/master/examples) + [link in top-level README](https://github.com/gotham-rs/gotham#learning).

There's nothing saying we _have_ to do it like them, but this format is what I'm familiar with.   

Also, I'll add a link to the documentation at the last line.

Reply:
Ah, sorry, I missed that you were pointing to the **examples**, not source code :) Yeah, that's good actually!

Reply:
I think we should do the same with all listed features.

Reply:
Updated, let me know your thoughts!

Reply:
Amazing! 👍 good to be merged